docs/setup.rst

```diff
@@ -807,6 +807,8 @@ configuring some options in paperless can help improve performance immensely:
     times. Thumbnails will be about 20% larger.
 *   If using docker, consider setting ``PAPERLESS_WEBSERVER_WORKERS`` to
     1. This will save some memory.
+*   Use the arm compatible docker-compose if you're wanting to use Tika on something like
+		a raspberry pi. The official apache/tika image doesn not support the arm architecture.
```

Comment:
Typo: “doesn”

Reply:
Not sure if this is a silly question, but how do I make changes to this PR? I've updated iwishiwasaneagle/paperless-ng:master [here](https://github.com/iwishiwasaneagle/paperless-ng/commit/cbb5199d5a1bca0d37cc09a8cb6b52ba7014f19e) but it's not showing up in this PR.

Reply:
Okay I tried "suggesting an edit" and it does not show up, neither is the commit my fork. So @s-martin, if you could propose the edit I'll accept it?

Reply:
I think the problem is your repo is forked https://github.com/jonaswinkler/paperless-ng while since #101 paperless-ngx is not forked from https://github.com/jonaswinkler/paperless-ng anymore.

unfortunately I don’t have enough GitHub skills to fix this. Maybe @qcasey can help?

Reply:
That's what I was thinking. Even committing my own suggestion to this PR only shows up in the original repo [jonaswinkler/paperless-ng@`bb3db0c` (#1354)](https://github.com/jonaswinkler/paperless-ng/pull/1354/commits/bb3db0cee12d4e67d9542d11ec6ece65ca95c867)